lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -75,13 +97,15 @@ class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
                   color: Colors.deepPurple,
                 ),
               ),
+              // set the selected feature layer source.
               value: _selectedFeatureLayerSource,
               icon: const Icon(
                 Icons.arrow_drop_down,
                 color: Colors.deepPurple,
               ),
               elevation: 16,
               style: const TextStyle(color: Colors.deepPurple),
+              // set the onChanged callback to update the selected feature layer source.
               onChanged: (String? featureLayerSource) {
                 setState(() {
                   _selectedFeatureLayerSource = featureLayerSource!;
```

Comment:
I think the `!` might be unnecessary. I think a null value here means the selection is being cleared, which is a valid state.